bindings/c/CMakeLists.txt

```diff
@@ -54,8 +54,8 @@ if(NOT WIN32)
     test/mako/mako.h
     test/mako/utils.c
     test/mako/utils.h
-    test/mako/zipf.c
-    test/mako/zipf.h)
+    ${PROJECT_SOURCE_DIR}/fdbclient/zipf.c
+    ${PROJECT_SOURCE_DIR}/fdbclient/zipf.h)
```

Comment:
The proper way to solve this is by linking against fdbclient (which I had assumed this code already did), rather than crossing module boundaries.

Reply:
The original idea was to have this C application as independent from FDB as possible, i.e. it was designed to work with any version of libfdb_c.so.  Zipfian distribution is a feature which should be inside the application itself, not the FDB client library, IMHO.  It seemed to me a bit hacky to have zipf stuff in fdbclient just for MakoWorkload.

Reply:
I kept the two files in `mako` cuz I used to think `fdbclient` is dynamically linked and I kept them to work with older libs. But it seems to be in `fdbclient.a`, which is statically linked. Will remove them.